mantidproject/mantid
# Reduce partial file loading in PD

It was observed that `AlignAndFocusPowderFromFiles` does a lot of work to load small amounts of files, but this generates overhead for "small" files. This PR adds checks to reduce the amount that an individual file is opened in the case of standard VULCAN reduction.

**To test:**

The current system tests should pass as well as the standard use cases for VULCAN (not in automated tests) should be better.

*There is no associated issue.*

*Does not need to be in the release notes* because it is a performance enhancement that most people won't see.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
